actors/builtin/power/power_actor.go

```diff
@@ -386,6 +390,14 @@ func (a Actor) processBatchProofVerifies(rt Runtime) {
 	res, err := rt.BatchVerifySeals(verifies)
 	builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to batch verify")
 
+	var rewret reward.ThisEpochRewardReturn
+	rewretcode := rt.Send(builtin.RewardActorAddr, builtin.MethodsReward.ThisEpochReward, nil, big.Zero(), &rewret)
+	builtin.RequireSuccess(rt, rewretcode, "failed to check epoch baseline power")
+
+	var pwr CurrentTotalPowerReturn
+	powretcode := rt.Send(builtin.StoragePowerActorAddr, builtin.MethodsPower.CurrentTotalPower, nil, big.Zero(), &pwr)
```

Comment:
Since you have access to state you can just do `st.ThisEpochQAPowerSmoothed` for what you want instead of calling to power actor.  I think you already did this in your follow on PR.

Reply:
can I just do this in the following PR and not worry about doing the work twice then having to merge it?